terraform/standalone/nms/gcp/variables.tf

```diff
@@ -43,6 +43,11 @@ variable "image_name" {
   type        = string
 }
 
+variable "network_name" {
+  description = "VPC network to be used"
+  type        = string
```

Comment:
should this default to `default` to match the previous behaviour?

Reply:
sure we can set the `default`